src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,13 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Check if the command mutates data in the address book
```

Comment:
A more precise phrasing is `Returns true if …` (instead of `Checks if …`)

Reply:
It is better to mention it returns true if the `Command` *potentially* mutates the data (header comments specify a contract, so try to be as precise as possible). E.g. Adding a duplicate person will not mutate data but the corresponding `AddCommand` object will still return `true` for this method.